server/src/db-client.ts

```diff
@@ -11,7 +12,10 @@ export function createDbClient(label: string): KnexType<any, unknown[]> {
   const instanceName = getHostInstance();
   function logPoolSize() {
     const poolSize = pool.numUsed() + pool.numFree();
-    logger.info(`${instanceName} ${label} DB pool size: ${poolSize}`);
+    logger.debug(`${instanceName} ${label} DB pool size: ${poolSize}`);
+    dogstatsd.gauge(`db.${label.toLowerCase()}.pool.size`, poolSize, [
+      `instance:${instanceName}`,
+    ]);
```

Comment:
The biggest question here is: is putting the “label” in the metric name and the instance as a tag correct, or should it be the other way around? Or should both be tags?

Reply:
🤷🏽 I'm not super familiar with Datadog, so not sure at all.

Reply:
No worries. I’ll go with what I did here, which is my gut instinct (I *think* this configuration makes it easiest to graph them against each other and to filter to a certain host, but am also no expert on DataDog).